DeepSpeech.py

```diff
@@ -484,7 +538,7 @@ def calculate_mean_edit_distance_and_loss(model_feeder, tower, dropout):
     avg_loss = tf.reduce_mean(total_loss)
 
     # Beam search decode the batch
-    decoded, _ = tf.nn.ctc_beam_search_decoder(logits, batch_seq_len, merge_repeated=False)
+    decoded, _ = decode_with_lm(logits, batch_seq_len, merge_repeated=False, beam_width=1024)
```

Comment:
Would it make sense to be able to keep the ability to use the vanilla tensorflow decoder ?

Reply:
I don't think anyone will want to use it, and I'd rather avoid forking the execution/maintenance paths and have some part of the code break because it's not being tested.